docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,162 @@
+
+== Team
+
+=== Communication channels
+
+* link:https://groups.google.com/d/forum/jenkinsci-dev[Jenkins Developer Mailing List]
+* Chat: N/A, to be created
+* Office hours: N/A, to be created
+
+=== Maintainer roles
+
+* Committer / reviewer
+* Core Pull Request Reviewer
+* Core Maintainer
+* Release Team Member
+
+**Committers** and **reviewers** are common contributors who submit their changes and who review changes/issues submitted by others.
+There is no special preconditions to do so, anyone is welcome to contribute.
+
+**Core Pull Request Reviewers** is the next step for contributors who are interested to review Jenkins pull requests on a regular basis.
+They get [Triage permissions](TODO) so that they can manage pull requests, request reviews and prepare changelog drafts in the pull request description.
+GitHub team: link:TODO[@jenkinsci/core-pr-reviewers].
+
+<!--TODO: consider switching to Admin-->
+**Core Maintainers** get `Write` permissions in the repository, and hence they are able to merge pull requests.
+Their responsibility is to perform reviews on a regular basis and to merge ready pull requests towards the weekly releases (`master` branch).
+They are also responsible to monitor the weekly release status and to perform triage of critical issues.
+GitHub team: link:TODO[@jenkinsci/core].
+
+**Release Team Members** are responsible for Jenkins weekly and LTS releases.
+Led by the link:TODO[Jenkins Release Officer], they initiate releases, prepare changelogs and backport changes into the link:TODO[Stable branch].
+Team members get `Admin` permissions in the Jenkins core repository, and they also get permissions to trigger release Pipelines.
+
+=== Ladder
+
+* Anyone can participate as a **committer** or as a **reviewer**.
+  There is no precondition except having a GitHub account, just submit pull requests or comment in existing ones!
+* **Core Pull Request Reviewers** can be nominated by contributors in the link:https://groups.google.com/d/forum/jenkinsci-dev[developer mailing list]. 
+Self-nomination are fine.
+Decision is made by a consensus in the mailing list or via voting at the link:https://jenkins.io/project/governance-meeting/[governance meeting].
+* **Core Maintainers** - same process as for Core PR reviewers.
+  All nominees must also sign an link:https://github.com/jenkinsci/infra-cla/[Individual Contributor License Agreement] before getting the permission in GitHub repositories.
+* **Release Team Members** are assigned by the Jenkins Release officer
+
+== Pull request review process
+
+Jenkins core is a mission-critical part of the ecosystem.
+We need to ensure that submitted pull requests are not only code complete,
+but also that they do not introduce undesired defects, breaking changes and technical debt.
+At the same time, we are interested to make the review process as simple as possible for contributors and maintainers.
+
+=== Code review goals
+
+* Ensure that the pull request is feasible.
```

Comment:
Don't unnecessarily write Powerpoint slides in text documents. This isn't for consumption by managers, we can handle actual paragraphs longer than one sentence.